pkg/operator/controller/controller.go

```diff
@@ -406,10 +406,7 @@ func (r *reconciler) ensureDNS(dns *operatorv1.DNS) error {
 		errs = append(errs, fmt.Errorf("failed to create ca bundle configmaps for dns %s: %w", dns.Name, err))
 	}
 
-	cmMap, err := r.caBundleRevisionMap(dns)
-	if err != nil {
-		return fmt.Errorf("failed to generate ca bundle revision map: %w", err)
-	}
+	cmMap := r.caBundleRevisionMap(dns)
```

Comment:
Can you comment as to how the rest of the code would work with a potentially empty cmMap?

Reply:
e.g., in https://github.com/openshift/cluster-dns-operator/pull/321/files#diff-6712a6f071bb599a25fa8780722861fd1f5cd051b22abd505c6fa7b9f3c7cb53R154

Reply:
Sure, I've added a comment.